articles/2016-09-01-semantic-html-tags.md

```diff
@@ -0,0 +1,98 @@
+# :name_badge: Semantic HTML Elements
+
+For this article, I assume you have a basic working knowledge of HTML. If this is not the case, read over a few of [these tutorials](http://htmldog.com/guides/html/beginner/), and then come back.
+
+### HTML5
+
+HTML5 was the newest iteration of of HTML published in 2014. This version primarily focused on enabling multiple devices and systems to interpret the markup languge. With a growing focus on responsive websites, screen readers and accessibility, the WWW introduced semantic tags. Specifically, they introduced new tags to represent new formats of media (`<canvas>`, `<video>`, `<audio>`), as well as ones to represent page structure (`<main>`, `<section>`, `<header>`, `<footer>`, `<nav>`, etc). For a full list of all available elements offered as of HTML 5, please refer to the [documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element).
+
+### Why are semantic tags better?
+
+The goal of semantic HTML elements is to better represent the content within. A `<div>` can represent just about any block-level element, but it is better to be more specific.
+
+#### Accessibility
+
+Screen readers, parsers, and search engines that try to read your HTML and attempt to understand its structure. If you use semantic tags, you are giving these systems an easier time. Your audience is diverse and has a wide range of abilities/needs that you must address by including [ARIA attributes](http://html5doctor.com/using-aria-in-html/) in your HTML code. These attributes are necessary to ensure that accessibility API's can get the relevant information about your page. As of HTML5, semantic elements have the ARIA *role* built in by default. While it is important to consider ARIA attributes and include them whenever possible, using the right element for the content is a good bare minimum.
```

Comment:
This first sentence isn't grammatically correct. Think it should be something more like "Screen readers, parsers, and search engines will try to read your HTML and attempt to understand its structure."